lib/advisor/core/questions/phrases_catalog.ex

```diff
@@ -1,52 +1,46 @@
 defmodule Advisor.Core.Questions.PhrasesCatalog do
-  alias Advisor.Core.Questions.PhraseKind
+  alias Advisor.Core.Questions.Phrase
 
   @path "lib/advisor/core/questions/questions.yml"
 
-  # TODO Maybe add a struct called `phrase`?
+  def find(ids) do
+    all()
+    |> flatten()
```

Comment:
Similar observation here. Flattening happens both here and in the `zip` function. Maybe we should revisit the data structure under question.